### DISCUSSION TITLE
Simplified likelihood from simplify tool

### DISCUSSION THREAD

**Discussion by sabinekraml** [scikit-hep/pyhf#1689] [Q&A] [answered]
Hi, this does not concern pyhf proper but, as the subject says, simplified json files produced with the simplify tool. I'm interested in knowing 

- why in the simplified likelihood the background expectations are replaced with the post-fit numbers; this eliminates any information about small deviations from SM expectations, relevant for e.g. BSM fits
- how correlated uncertainties are treated in simplify: if there are both correlated and uncorrelated nuisances, how are the final region-to-region correlations determined? The mathematical description would be helpful. 

Thanks! Sabine

**Comment by matthewfeickert** [comment, marked as answer]
@sabinekraml As these questions are about `simplify` I've transferred this over to `simplify`'s repo as https://github.com/eschanet/simplify/issues/19 where it can be better answered.
